spec.md

```diff
@@ -602,6 +602,12 @@ This RPC will be called by the CO to provision a new volume on behalf of a user
 This operation MUST be idempotent.
 If a volume corresponding to the specified volume `name` already exists and is compatible with the specified `capacity_range`, `volume_capabilities` and `parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
 
+Plugins MAY create 3 types of volumes:
+
+- Empty volumes. When plugin supports `CREATE_DELETE_VOLUME` capability.
```

Comment:
nit: the next two are "optional capability" but this is simply "capability" -- but it's also optional. why the difference?

Reply:
My bad, I'll update it.